### ISSUE TITLE
Package loading warning after updating 9.1

### ISSUE THREAD

**Issue by sepw** [Quick/Nimble#881]
### What did you do?

Updated SPM package dependencies, which pulled in Nimble 9.1. I was previously using 9.0.1.

### What did you expect to happen?

I expected Nimble to build without errors or warnings.

### What actually happened instead?

Xcode (12.5) can build Nimble successfully but presents a package loading warning for Nimble:

```
Showing Recent Messages

Nimble
found 1 file(s) which are unhandled; explicitly declare them as resources or exclude from the target
    /Users/me/Library/Developer/Xcode/DerivedData/Project-axynmiysuypgzucopudqyvqlpcbk/SourcePackages/checkouts/Nimble/Sources/Nimble/Info.plist
```

The issues appears to relate to the inclusion/exclusion of resources from the Nimble SPM package manifest.

### Environment

List the software versions you're using:

- Nimble: 9.1
- Xcode Version: 12.5
- Swift Version: Xcode Default

Please also mention which package manager you used and its version. Delete the
other package managers in this list:

- Swift Package Manager 5.4.0

### Project that demonstrates the issue

Create a new SPM library project, add a dependency for Nimble with dependency:

```swift
dependencies: [
  .package(url: "https://github.com/Quick/Nimble.git", from: "9.0.0"),
]
```

**Comment by ikesyo** [comment]
This is already addressed in master by #878 but we would backport to 9.x.

**Comment by ikesyo** [comment]
[v9.2.0](https://github.com/Quick/Nimble/releases/tag/v9.2.0) has been released.
